### PR TITLE
[2108] [7802803] New resource type name to scan everything except pipelines

### PR DESCRIPTION
**Pull request by juhi037**
## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.ADO/Framework/Abstracts/MetaInfoProvider.ps1 [hunk 1]
```diff
@@ -246,22 +246,22 @@ class MetaInfoProvider {
 			}
 		}
 
-		if ($ResourceTypeName -in ([ResourceTypeName]::ServiceConnection, [ResourceTypeName]::All, [ResourceTypeName]::Build_Release_SvcConn_AgentPool_VarGroup_User))
+		if ($ResourceTypeName -in ([ResourceTypeName]::ServiceConnection, [ResourceTypeName]::All, [ResourceTypeName]::Build_Release_SvcConn_AgentPool_VarGroup_User, [ResourceTypeName]::SvcConn_AgentPool_VarGroup_CommonSVTControls))
```
Comment (arvindsingh-cloud) [inline src/AzSK.ADO/Framework/Abstracts/MetaInfoProvider.ps1:249]: CommonSVT/CommonSVTResources

File: src/AzSK.ADO/Framework/Helpers/SVTMapping.ps1 [hunk 2]
```diff
@@ -104,4 +104,4 @@ class SVTMapping
 	)
 }
 
-Invoke-Expression "enum ResourceTypeName { `r`n All `r`n $([SVTMapping]::GetResourceTypeEnumItems())`r`n Org_Project_User `r`n Build_Release `r`n Build_Release_SvcConn_AgentPool_VarGroup_User `r`n}";
+Invoke-Expression "enum ResourceTypeName { `r`n All `r`n $([SVTMapping]::GetResourceTypeEnumItems())`r`n Org_Project_User `r`n Build_Release `r`n Build_Release_SvcConn_AgentPool_VarGroup_User `r`n SvcConn_AgentPool_VarGroup_CommonSVTControls `r`n}";
```
Comment (arvindsingh-cloud) [inline src/AzSK.ADO/Framework/Helpers/SVTMapping.ps1:107]: CommonSVT/CommonSVTResources ??

File: src/AzSK.ADO/Framework/Abstracts/MetaInfoProvider.ps1 (hunk 1 above)
Comment (juhi037) [reply src/AzSK.ADO/Framework/Abstracts/MetaInfoProvider.ps1]: changed

File: src/AzSK.ADO/Framework/Helpers/SVTMapping.ps1 (hunk 2 above)
Comment (juhi037) [reply src/AzSK.ADO/Framework/Helpers/SVTMapping.ps1]: changed

**Review by arvindsingh-cloud — APPROVED** [review]

**Review by SaikumarRagala — APPROVED** [review]

**Pull request merged**